sssd_test_framework/hosts/ldap.py

```diff
@@ -117,6 +119,8 @@ def restore(self, backup_data: Any | None) -> None:
         if backup_data is None:
             return
 
+        self.logger.info("Restoring LDAP server from memory")
```

Comment:
from memory? I don't quite understand this part.

Reply:
LDAP backup is stored into memory - into python variable. Other hosts are storing it into a file.